[enocean] Nullpointer because “eepType” is null

Just upgraded from OH 4.2.3 to 4.3.1 and got problems with EnOcean binding.

Just for my measurement switches I got the following error message. On the other side mechanical handles works fine without any error. Btw, switching and measurements are done as expected.

```
2025-01-03 10:58:17.174 [TRACE] [ernal.transceiver.EnOceanTransceiver] - Received Sync Byte

2025-01-03 10:58:17.176 [TRACE] [ernal.transceiver.EnOceanTransceiver] - >> Received header, data length 10 optional length 7 packet type 1

2025-01-03 10:58:17.180 [DEBUG] [ernal.transceiver.EnOceanTransceiver] - RADIO_ERP1 with RORG _4BS for xxxxxxxx payload A50000510Cxxxxxxxx0001FFFFFFFF5200 received

2025-01-03 10:58:17.181 [ERROR] [ernal.transceiver.EnOceanTransceiver] - Exception in informListeners

java.lang.NullPointerException: Cannot invoke “org.openhab.binding.enocean.internal.eep.EEPType.getEEPClass()” because “eepType” is null

at org.openhab.binding.enocean.internal.eep.EEPFactory.buildEEP(EEPFactory.java:63) ~[bundleFile:?]

at org.openhab.binding.enocean.internal.handler.EnOceanBaseSensorHandler.packetReceived(EnOceanBaseSensorHandler.java:159) ~[bundleFile:?]

at org.openhab.binding.enocean.internal.transceiver.EnOceanTransceiver.lambda$0(EnOceanTransceiver.java:372) ~[bundleFile:?]

at java.lang.Iterable.forEach(Iterable.java:75) ~[?:?]

at org.openhab.binding.enocean.internal.transceiver.EnOceanTransceiver.informListeners(EnOceanTransceiver.java:372) [bundleFile:?]

at org.openhab.binding.enocean.internal.transceiver.EnOceanESP3Transceiver.processMessage(EnOceanESP3Transceiver.java:149) [bundleFile:?]

at org.openhab.binding.enocean.internal.transceiver.EnOceanTransceiver.receivePackets(EnOceanTransceiver.java:305) [bundleFile:?]

at org.openhab.binding.enocean.internal.transceiver.EnOceanTransceiver$1.run(EnOceanTransceiver.java:230) [bundleFile:?]

at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539) [?:?]

at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]

at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) [?:?]

at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]

at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]

at java.lang.Thread.run(Thread.java:840) [?:?]
```
My Environment
* Version used: openHAB 4.3.1 Release Build Main UI Commit a695375f
* Operating System and version: Raspbian GNU/Linux 11 (bullseye) on Raspberry Pi 4 Model B Rev 1.4

--- a/bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/handler/EnOceanBaseSensorHandler.java
+++ b/bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/handler/EnOceanBaseSensorHandler.java
@@ -18,6 +18,7 @@ import java.util.Collection;
 import java.util.Collections;
 import java.util.Comparator;
 import java.util.Hashtable;
+import java.util.Objects;
 import java.util.Set;
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
@@ -87,7 +88,7 @@ public class EnOceanBaseSensorHandler extends EnOceanBaseThingHandler implements
             config.receivingEEPId.forEach(receivingEEP -> {
                 EEPType receivingEEPType = EEPType.getType(receivingEEP);
                 EEPType existingKey = receivingEEPTypes.putIfAbsent(receivingEEPType.getRORG(), receivingEEPType);
-                if (existingKey != null) {
+                if (!Objects.isNull(existingKey)) {
                     throw new IllegalArgumentException("Receiving more than one EEP of the same RORG is not supported");
                 }
             });
@@ -155,6 +156,9 @@ public class EnOceanBaseSensorHandler extends EnOceanBaseThingHandler implements
         ERP1Message msg = (ERP1Message) packet;
 
         EEPType localReceivingType = receivingEEPTypes.get(msg.getRORG());
+        if (Objects.isNull(localReceivingType)) {
+            return;
+        }
 
         EEP eep = EEPFactory.buildEEP(localReceivingType, (ERP1Message) packet);
         logger.debug("ESP Packet payload {} for {} received", HexUtils.bytesToHex(packet.getPayload()),

